docs/data/data-grid/editing/editing.md

```diff
@@ -174,6 +174,26 @@ The value returned is used later as an argument on a call to `apiRef.current.upd
 />
 ```
 
+If you want to delete a row from the internal state of the Data Grid, you can return an additional property `_action: 'delete'` in the row object from the `processRowUpdate` callback.
+This will remove the row from the internal state of the Data Grid.
```

Comment:
I'd like to explain a bit about the incentive of using the processRowUpdate for deletion. I think most people manipulate `rows` state for the purpose, so we need to repeat this information in docs to promote the usage of `updateRows` more.
```suggestion
If you want to delete a row from the internal state of the Data Grid, you can return an additional property `_action: 'delete'` in the row object from the `processRowUpdate` callback. This will remove the row from the internal state of the Data Grid.
It is a more performant way to delete a row as compared to updating the [`rows` prop](/x/react-data-grid/row-updates/#the-rows-prop) or using `setRows` API method because processRowUpdate uses the [`updateRows`](https://mui.com/x/react-data-grid/row-updates/#the-updaterows-method) under the hood which doesn't cause a full regeneration of the row tree.
```